tox.ini

```diff
@@ -7,13 +7,12 @@ setenv =
 deps =
     flake8
     pytest
-    pytest-cov
     pytest-xdist
     mock
 
 commands=
     flake8 releasetasks
-    py.test --verbose -n8 --cov=releasetasks --cov-report term-missing --doctest-modules releasetasks
```

Comment:
It's always 100%, because we have only one function to call. So it's kind of useless and doesn't represent our real coverage of the templates.